cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

```diff
@@ -92,6 +116,14 @@ It is defined as such:
 * `POINT` indexes are used over `RANGE` and `TEXT` indexes for distance and within a bounding box.
 * `RANGE` indexes are preferred over `TEXT` and `POINT` indexes in all other cases.
 
+`TOKEN LOOKUP` indexes are not defined in this order since they never solve the same set of predicates as the other indexes.
+
+include::ql/administration/indexes/node-label-lookup-index-example.asciidoc[leveloffset=2]
+
+include::ql/administration/indexes/relationship-type-lookup-index-example.asciidoc[leveloffset=2]
+
+include::ql/administration/indexes/relationship-range-index-example.asciidoc[leveloffset=2]
```

Comment:
This is included twice...

`include::ql/administration/indexes/relationship-range-index-example.asciidoc[leveloffset=2]`